hack/jenkins/e2e.sh

```diff
@@ -215,18 +266,43 @@ export KUBE_GCE_ZONE=${E2E_ZONE}
 export KUBE_GCE_NETWORK=${E2E_NETWORK}
 export KUBE_GCE_INSTANCE_PREFIX=${KUBE_GCE_INSTANCE_PREFIX:-}
 export KUBE_GCS_STAGING_PATH_SUFFIX=${KUBE_GCS_STAGING_PATH_SUFFIX:-}
+export KUBE_GCE_MINION_PROJECT=${KUBE_GCE_MINION_PROJECT:-}
```

Comment:
I remember myself added these lines for Trusty release jobs, which are now moved to branch 1.1. Don't know if you still want them for 1.0.